### ISSUE TITLE
[4.2.8] Removing one association from an article remove all language associations

### ISSUE THREAD

**Issue by Flowman** [joomla/joomla-cms#39982]
### Steps to reproduce the issue

Create an multilingual site with minimum 3 languages. 
Create 3 articles and associate all languages.
Remove one association.

![issue](https://user-images.githubusercontent.com/212680/222433344-a1ef026d-2c86-45db-90cd-bd1aa86547fd.gif)

### Expected result

Only the selected association should be removed.

### Actual result

All associations for the selected article is removed.

### System information (as much as possible)

Database Type 	mysql
Database Version 	8.0.29-21
PHP Version 	8.0.24
Joomla! Version 	Joomla! 4.2.8 Stable [ Uaminifu ] 16-February-2023 15:00 GMT

**Comment by wojtekxtx** [comment]
@Flowman strange. Cannot be that (by mistake) you selected *all 3* associations?

**Comment by Flowman** [comment]
Check the video clip. Those it look like I clear more then one? I remove the Swedish and it remove the English GB as a bonus.

**Comment by Flowman** [comment]
If you check the video clip, you can see I remove the Australian language, But as a bonus it remove the Swedish too from the Australian for no reason.

**Comment by krishnagandhicode** [comment]
> @krishnagandhicode just came across this https://github.com/joomla/joomla-cms/issues/39982 is it the same thing as fixed here?

yes @brianteeman  this is around the same area as my PR (#48208), but what I fixed there was the association breaking for categories. This one is reported for articles and **Articles are not broken.**

going by the video in the desc: the three articles are one group, **not three pairs**. Before the save the group is English + Australian + Swedish. Removing Australian from the English article and saving leaves the group as English + Swedish, and the Australian article is simply no longer in it.

So it has not separately lost its link to Swedish. It has left the group, and Swedish was only ever in that group. English and Swedish are still associated afterwards, as the association column confirms.

So I believe this is working as designed, and the issue can/should be closed.

**Comment by CSGoat0** [comment]
Thanks @Flowman for the report, I will close this issue for now. You may read this [comment](https://github.com/joomla/joomla-cms/issues/39982#issuecomment-5455484341) and if you think this is a mistake please reply to it.
